shared/applicability/partition-var-log-audit.yml

```diff
@@ -0,0 +1,5 @@
+name: "cpe:/a:partition-var-log-audit"
```

Comment:
Instead of creating new platforms, I would propose consolidating the existing `partition-tmp` and `partition-var-tmp` into a single templated `partition` platform. Then, it would be easier to define platforms for any other partitions. Having a templated platform would reduce code duplication.

Reply:
OK sure, can you help on doing that (I'm just good with copy/pasting :-))